docs/i18n.rst

```diff
@@ -275,7 +248,7 @@ web interfaces, these need to be created:
 
 .. code:: sh
 
-    $ securedrop/bin/dev-shell ./i18n_tool.py --verbose translate-messages --compile
+    $ securedrop/bin/dev-shell make extract-strings
```

Comment:
This section describes the `.po`→`.mo` compilation rather than the `.py`→`.po[t]` extraction.  But it can go away too: if we're not letting Weblate compile our `.mo` files, it'll be part of `make dev` and/or `dev-deps` (TBD).